web/src/components/storage/SpaceActionsTable.jsx

```diff
@@ -122,16 +120,45 @@ const DeviceAction = ({ device, action, isDisabled = false, onChange }) => {
       }
     >
       <FormSelectOption value="force_delete" label={_("Delete")} />
-      {/* Resize action does not make sense for drives, so it is filtered out. */}
-      <If
-        condition={!device.isDrive}
-        then={<FormSelectOption value="resize" label={_("Allow resize")} />}
-      />
+      <FormSelectOption value="resize" label={_("Allow resize")} />
       <FormSelectOption value="keep" label={_("Do not modify")} />
     </FormSelect>
   );
 };
 
+/**
+ * Column content with the space action (a form or a description) for a device
+ * @component
+ *
+ * @param {object} props
+ * @param {StorageDevice} props.device
+ * @param {string} props.action - Possible values: "force_delete", "resize" or "keep".
+ * @param {boolean} [props.isDisabled=false]
+ * @param {(action: SpaceAction) => void} [props.onChange]
+ */
+const DeviceAction = ({ device, action, isDisabled = false, onChange }) => {
+  if (!device.sid) return null;
+
+  if (device.type === "partition") {
+    return (
+      <DeviceActionForm
+        device={device}
+        action={action}
+        isDisabled={isDisabled}
+        onChange={onChange}
+      />
+    );
+  }
+
+  if (device.filesystem || device.component)
+    return _("The content may be deleted");
+
+  if (!device.partitionTable || device.partitionTable.partitions.length === 0)
+    return _("No content found");
```

Comment:
NP: I still thinking that "No content found" is a bad content for an "Action" column. But I do not have a good proposal at the time of writing.